### PR TITLE
Integrated Codacy & Updated Documentation

### PR DESCRIPTION
**Pull request by SachinRammoorthy**
Issue: #46 
Also added codacy grade badge to README. However, this corresponds to my fork of this project. Please enable Codacy and I can then change the badge to correspond to the actual repo.

Documentation also updated

screen shot 2017-12-24 at 8 50 32 pm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # JBoss Blog
 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/e4e0a822a35342459006c797fa156ccf)](https://www.codacy.com/app/SachinRammoorthy/blog.jboss-outreach.org?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=SachinRammoorthy/blog.jboss-outreach.org&amp;utm_campaign=Badge_Grade)
```
Comment (aashutoshrathi) [inline README.md]: Won't this badge show status for you fork?

Comment (SachinRammoorthy) [reply README.md]: Yes, that is why I asked for codacy to be enabled on the first comment and on my task so I can link it to the actual one...

Comment (aashutoshrathi) [reply README.md]: Oh, I missed that PR comment, Let's give the org admin a call for this.
@garg-anuj Check this we need to activate codacy for this.

**Comment by SachinRammoorthy** [comment]
Can codacy be enabled for this repo please?

**Comment by aashutoshrathi** [comment]
I have added Codacy for this repository, please update badge with
`[Codacy Badge](https://www.codacy.com/app/aashutoshrathi/blog.jboss-outreach.org?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=jboss-outreach/blog.jboss-outreach.org&amp;utm_campaign=Badge_Grade)`

i.e. 
[Codacy Badge](https://www.codacy.com/app/aashutoshrathi/blog.jboss-outreach.org?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=jboss-outreach/blog.jboss-outreach.org&amp;utm_campaign=Badge_Grade)

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi replaced the badge.

**Pull request merged**